sympy/physics/continuum_mechanics/beam.py

```diff
@@ -210,6 +215,16 @@ def reaction_loads(self):
         """ Returns the reaction forces in a dictionary."""
         return self._reaction_loads
 
+    @property
+    def rotation_jumps(self):
+        """ Returns the rotation jumps in hinges multiplied by the elastic modulus and second moment in a dictionary."""
+        return self._rotation_jumps
```

Comment:
@mvg2002 Could you make clear that the rotation jump include EI in the variable name itself?